ksql-common/src/main/java/io/confluent/ksql/processing/log/ProcessingLogMessageSchema.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.processing.log;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+public final class ProcessingLogMessageSchema {
+  private ProcessingLogMessageSchema() {
+  }
+
+  private static final String NAMESPACE = "io.confluent.ksql.processing.log.";
+
+  public enum MessageType {
```

Comment:
you can define a `getSchema` (etc...) methods defined on `MessageType` to have a stronger coupling than a constant that shares the name.